nginx/nginx-dropin-iml.conf

```
@@ -0,0 +1,4 @@
+[Service]
+EnvironmentFile=/var/lib/chroma/iml-settings.conf
+EnvironmentFile=-/var/lib/chroma/overrides.conf
+ExecStartPre=/usr/bin/iml-config nginx generate-config --output /etc/nginx/conf.d/chroma-manager.conf --path /usr/share/chroma-manager/chroma-manager.conf.template
```

nit: switch path and output argument order

```suggestion
ExecStartPre=/usr/bin/iml-config nginx generate-config --path /usr/share/chroma-manager/chroma-manager.conf.template --output /etc/nginx/conf.d/chroma-manager.conf
```